Makefile

```diff
@@ -116,8 +116,12 @@ clean-extensions:
 clean-npm:
 	rm -rf src/extensions/npm/extensions/{dist,__mocks__,node_modules}
 
+.PHONY: clean-extension-api
+clean-extension-api:
+	rm -rf src/extensions/**/*.d.ts
+
 .PHONY: clean
-clean: clean-npm clean-extensions
+clean: clean-npm clean-extensions clean-extension-api
```

Comment:
Nit: I don't understand this change :( Better commit message in the future? :)

Reply:
I don't remember why I added this as well and it doesn't look required. Will remove.